src/App.css

```diff
@@ -0,0 +1,15 @@
+/* addind margin to avoid content to go under sidebar */
+.App {
+  margin-left: 50px; /* Same as the width of the sidebar */
```

Comment:
I would probably hide the sidebar or display it differently on mobile since the space is scarce:

![Skärmavbild 2024-03-13 kl  11 00 44](https://github.com/Technigo/project-music-releases-vite/assets/51529201/a76194f0-2d6b-4993-9c2d-b6c39cd22fc5)